cypress/integration/finance/ledgers/ledgers.create.spec.js

```diff
@@ -0,0 +1,25 @@
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import TopMenu from '../../../support/fragments/topMenu';
+
+describe('ui-finance: Ledger list search and filters', () => {
+  const timestamp = (new Date()).getTime();
+
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'))
+      .then(() => {
+        cy.visit(TopMenu.ledgerPath);
+      });
+  });
+
+  it('C4053 should create new ledger if mandatory fields are filled', function () {
```

Comment:
can't see implementations of:
"Ledgers filters are displayed (status, acquisition unit)"
"'Choose a filter or enter a search query to show results' message is displayed at the right part of the screen"
may be you should discuss with PO and remove these expectation from TestRail